os_dep/linux/usb_intf.c

```diff
@@ -70,6 +70,7 @@ static void rtw_dev_remove(struct usb_interface *pusb_intf);
 #define USB_VENDER_ID_REALTEK		0x0BDA
 
 /* DID_USB_v915_20121224 */
+
```

Comment:
Looks like you accidentally added an extra space here. Don't worry about it though.